packages/world/gas-report.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "source": "test/KeysInTableModule.t.sol",
+    "name": "compute the target table selector",
+    "functionCall": "bytes32 targetTableSelector = getTargetTableSelector(MODULE_NAMESPACE, sourceTableId)",
+    "gasUsed": 2240
+  },
```

Comment:
should we add another gas report for setting a value on a table with this module installed, and for getting the keys from a table? (Should be as easy as adding the `!gasreport` comment in the existing tests in `KeysInTableModule.t.sol`)

Reply:
Okay figured out why the other gas reports don't show up here: it's because forge doesn't print logs for fuzz tests (gas-report relies on console logs to parse the gas usage) and this is the only non-fuzz test. I think fuzz tests are nice, but it would also be useful to track the gas cost of this module for optimizations in the future, so maybe we should add a couple non-fuzz test instances for gas-report purposes?

Reply:
Thanks for figuring this out, it stumped me. I will add a couple normal tests.